bioloid/bus.py

```diff
@@ -200,7 +199,7 @@ def send_write(self, dev_id, offset, data, deferred=False):
             else:
                 log('Sending {} to ID {} offset 0x{:02x} len {}'.format(cmd_str, dev_id, offset, len(data)))
         cmd = packet.Command.REG_WRITE if deferred else packet.Command.WRITE
-        pkt_data = bytearray(len(data))
+        pkt_data = bytearray(len(data + 1))
         pkt_data[0] = offset
         pkt_data[1:] = data
```

Comment:
The `pkt_data[1:]` was previously dynamically adding 1 to the bytearray length.  But it worked fine.  I don't know enough micropython interals to assert that this is in fact faster, but I think it might be?